packages/create-next-app/helpers/git.ts

```diff
@@ -30,24 +30,26 @@ function isDefaultBranchSet(): boolean {
 export function tryGitInit(root: string): boolean {
   let didInit = false
   try {
-    execSync('git --version', { stdio: 'ignore' })
+    execSync('git --version', { stdio: 'inherit' })
```

Comment:
I feel like we omitted this on purpose. We should have access to those but not with the default log level. Do we have a configurable log level?

Reply:
All these changes are only temporary and part of the [Trying to find out why there's not git repo created in the CI](https://github.com/vercel/next.js/pull/64826/commits/c19acba4bb096329137034b86f2715ad28adb338) commit. I will revert them when I've fixed the git CI issue.

Reply:
Will convert to a draft until then...